ex9/lib/main.dart

```diff
@@ -0,0 +1,113 @@
+import 'package:flutter/material.dart';
+import 'package:cloud_firestore/cloud_firestore.dart';
+
+void main() => runApp(MaterialApp(home: FirestoreFirstDemo(),));
+
+FirestoreFirstDemoState pageState;
+
+class FirestoreFirstDemo extends StatefulWidget {
+  @override
+  FirestoreFirstDemoState createState() {
+    pageState = FirestoreFirstDemoState();
+    return pageState;
+  }
+}
+
+class FirestoreFirstDemoState extends State<FirestoreFirstDemo> {
+  final firestoreInstance =  Firestore .instance;
```

Comment:
Please use VSC's code formatting tool to fix spacing. It should be `Firestore.instance`, not `Firestore .instance`. See this [slide](https://docs.google.com/presentation/d/1tybmc0YPWrdmRy72DM4TAvJq9HPVocZgZB-k9tEdNRo/edit#slide=id.g855f226bc4_9_17).